### PR TITLE
Image Service - Minor refinements

### PR DESCRIPTION
**Pull request by irwin-r**

### CODE REVIEW

File: packages/images/src/service.js [hunk 1]
```diff
@@ -122,22 +122,23 @@ export class ImageService {
...
-            await sharpImage.resize(resizeOptions);
-          }
-          await sharpImage.toFile(transformedFilename);
-        } catch (err) {
-          console.error(err);
-          return res.status(500).send({ error: 'Internal server error' });
-        }
+      let transformedFilename = nodePath.join(this.transformsPath, `${id}${suffix}${format}`);
+
+      // If a cached copy of the transformed file exists, don't re-create it
```
Comment (JedWatson) [inline packages/images/src/service.js]: So... I'm pretty sure it introduces an error.

The previous logic was:

- if the cached image doesn't exist, generate it
- either way, send the image from the cache

Unless I'm misinterpreting the new code here, if a new image is created, it won't be sent

Comment (JedWatson) [reply packages/images/src/service.js]: Oh also, please check against the latest commits, there's a merge conflict (I think we both may have made similar changes concurrently based on your review of my PR, but this PR does include improvements I didn't do so would be great to merge it)

Comment (irwin-r) [reply packages/images/src/service.js]: Good catch -- not sure how I missed that in the first place 😭

**Comment by bladey** [comment]
Thanks for this PR @irwin-r.

**Keystone 5** has officially moved into active maintenance mode as we push towards the next major new version **Keystone Next**, you can find out more information about this transition [here](https://github.com/keystonejs/keystone/issues/4962).

In an effort to sustain the project going forward, we're cleaning up and closing old PRs such as this one.

However, you are welcome to create a PR under the [Keystone 5](https://github.com/keystonejs/keystone-5/pulls) repository.

Please let me know if you have any questions.

**Pull request closed without merging**